src/test/java/org/folio/api/AllowedServicePointsApiTest.java

```diff
@@ -124,9 +147,218 @@ void allowedServicePointsShouldReturn422WhenParametersAreInvalid() {
       .expectStatus().isEqualTo(422);
   }
 
+  @Test
+  void replaceForRequestLinkedToItemWhenPrimaryRequestTypeIsAllowedInBorrowingTenant() {
+    createEcsTlr(true);
+
+    var mockAllowedSpResponseFromBorrowingTenant = new AllowedServicePointsResponse()
+      .page(Set.of(buildAllowedServicePoint("borrowing-page")))
+      .hold(Set.of(buildAllowedServicePoint("borrowing-hold")))
+      .recall(Set.of(buildAllowedServicePoint("borrowing-recall")));
+
+    wireMockServer.stubFor(get(urlMatching(ALLOWED_SERVICE_POINTS_MOD_CIRCULATION_URL +
+      String.format("\\?operation=replace&requestId=%s&useStubItem=true", PRIMARY_REQUEST_ID)))
```

Comment:
What does `\\` do?

Reply:
It escapes `?`, which without escaping means "any character" in regex.

Reply:
It escapes the `?`, which has a special meaning in regex